### DISCUSSION TITLE
skip scanning imported packages

### DISCUSSION THREAD

**Discussion by jvrahav** [golangci/golangci-lint#1916] [Q&A] [answered]
Is there a way to skip scanning/validating packages being imported in a file. 

There are third party components that are importing in my file and golangci-lint reports error in the one of the files in the thrid party component.

this blocks running lint on my application. Any suggestions for this would be great

**Comment by Antonboom** [comment, marked as answer]
Hi!

Ignore third-party components paths with `skip-dirs` or `skip-files`.
https://golangci-lint.run/usage/configuration/
